### ISSUE TITLE
Extending the extenisons

### ISSUE THREAD

**Issue by imranebs** [Microsoft/AL#2894]
Is there a way to extend a table in another extension that is installed on the server?

**Comment by SUD-D365** [comment]
@imranebs  yes you can, by defining the **Dependencies** parameter in App.json file.

![image](https://user-images.githubusercontent.com/33481576/42744191-350cdce0-88e8-11e8-9bfa-836def459cfc.png)

**Comment by imranebs** [comment]
How do we update the symbols to get the object names from the extensions?

**Comment by SUD-D365** [comment]
@imranebs  once you defined the dependencies in App.json, it will automatically download the symbols from that installed extension , and you will be able to extend objects from installed extension.

**Comment by imranebs** [comment]
Does not work for me. Do I need any other options set?

**Comment by thpeder** [comment]
Hi @imranebs,
You do as SUD-D365 said, specifying the other extensions information under dependencies in the app.json file.
Then in VSCode you go and download the symbols, assuming that the extension that you are depending on is already published to the server. You can check that by going to the Extension Management page, or if you run your own server then you can also use the mangement interface in Powershell with the command [Get-NAVAppInfo](https://docs.microsoft.com/en-us/powershell/module/microsoft.dynamics.nav.apps.management/get-navappinfo).
